Dockerfile

```diff
@@ -1,24 +1,20 @@
-FROM rocker/tidyverse
+FROM hnskde/qmongr-base-r
 
 LABEL maintainer "Are Edvardsen <are.edvardsen@helse-nord.no>"
 LABEL com.centurylinklabs.watchtower.enable="true"
 
-# system libraries of general use
-RUN apt-get update && apt-get install -y \
-    sudo \
-    pandoc \
-    pandoc-citeproc \
-    libcurl4-gnutls-dev \
-    libcairo2-dev \
-    libxt-dev \
-    libxml2-dev \
-    libssl-dev
-
-# basic R functionality
-RUN R -e "install.packages(c('remotes'), repos='https://cloud.r-project.org/')"
-
-# install qmongr app
-RUN R -e "remotes::install_github('SKDE-Felles/qmongr', upgrade = 'never')"
+
+# install package dependency from github
+RUN R -e "remotes::install_github('SKDE-Felles/qmongrdata', upgrade = 'always')"
```

Comment:
Can 
```
remotes::install_github('SKDE-Felles/qmongrdata', upgrade = 'always')
```
be replaced by something like
```
devtools::install_deps(".")
```
?

Reply:
In fact, the sole purpose is to make sure the latest version of _qmongrdata_ will be present when building the _qmongr_ docker image. Everything else should be taken care of by the [initial docker image](https://github.com/SKDE-Felles/qmongr-base-r/blob/master/Dockerfile). To ensure a shortest possible build time _upgrade_ should be set to 'never' here as well